js/src/main/boilerplate/spray/json/ProductFormatsInstances.scala.template

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2011,2012 Mathias Doenitz, Johannes Rudolph
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package spray.json
+
+trait ProductFormatsInstances { self: ProductFormats with StandardFormats =>
```

Comment:
If those are exactly the same could we maybe just symlink them instead?

Reply:
They're almost the same but not quite: the JS version doesn't include the jsonFormatX template `def jsonFormat1[[#P1 :JF#])`